wpt

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
This breaks macOS, which doesn't ship with any `python2`.

Reply:
Oh ok. Too bad.